docs/source/starter/core_guide.rst

```diff
@@ -1139,7 +1139,6 @@ spread all over files.
     # transforms
     train_transforms = ...
     val_transforms = ...
-    test_transforms = ...
```

Comment:
This line doesn't need to be deleted, I think?

Reply:
no.